src/devices/Directory.Build.props

```diff
@@ -3,6 +3,7 @@
   <Import Project="../../eng/Versions.props" />
   <Import Project="../../eng/Analyzers.props" />
   <PropertyGroup>
+    <TargetFrameworksBase>net5.0;netcoreapp2.1;net48;</TargetFrameworksBase>
```

Comment:
I don't think that adding a net48 configuration is the right thing to do here, instead we should try to add a netstandard2.0 configuration to all projects that support it, and if the build works fine, we can use that instead of the netcoreapp2.1 configuration in case there are no differences between netcoreapp2.1 and netsandard2.0. We already started doing this a while back, but the PR was never completed so perhaps this is a good oportunity to take care of it.

Main advantage of using netstandard2.0 as opposed to net48, is that with it it will be actually making the bindings runnable accross other platforms like Mono, Xamarin, and UWP which will probably be more handy than a net48 configuration.

Reply:
I have tried that, but it didn't work. But based on what I have done now, it seems it could actually work. Maybe I'll need your help in getting the packaging stuff right - I can get it to compile, but the way the main Iot.Device.Bindings.dll is built gives me a bit of a headache.

Reply:
Some time ago I merged a PR in Iot.Device.Bindings that will basically get the best target framework for all of the devices for each TFM that Iot.Device.Bindings compiles for, and will try to build  those particular sources for it so in theory this shouldn't be to hard to do now. Work will mostly involve adding netstandard2.0 configs on all devices that support it, and add a netstandard2.0 config to iot.device.bindings. Finally we will just have to add some exclusions into iot.device.bidnings for those devices that won't support netstandard2.0.